### PR TITLE
adding test coverage for module stream filter cli

### PR DESCRIPTION
**Pull request by omkarkhatavkar**
### Test Result 

```
Testing started at 5:34 PM ...
/home/okhatavk/okhatavk/Satellite/robottelo/env/bin/python /home/okhatavk/Downloads/pycharm-community-2018.1.4/helpers/pycharm/_jb_pytest_runner.py --target test_contentview.py::ContentViewTestCase.test_positive_add_module_stream_filter_rule
Launching py.test with arguments test_contentview.py::ContentViewTestCase::test_positive_add_module_stream_filter_rule in /home/okhatavk/Satellite/robottelo/tests/foreman/cli

============================= test session starts ==============================
platform linux -- Python 3.7.2, pytest-4.6.3, py-1.8.0, pluggy-0.13.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/okhatavk/Satellite/robottelo
plugins: cov-2.8.1, xdist-1.30.0, mock-1.10.4, services-1.3.1, forked-1.1.12019-12-11 12:04:12 - conftest - DEBUG - Collected 1 test cases
2019-12-11 17:34:12 - robottelo.helpers - DEBUG - Generated file bz_cache.json with BZ collect data
collected 1 item

test_contentview.py                                                     [100%]

========================== 1 passed in 171.91 seconds ==========================2019-12-11 17:34:12
```

### CODE REVIEW

**Comment by latran** [comment]
What's the benefit of having this test case versus the one we already use in ui automation? https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/ui/test_contentview.py#L2650

**Comment by omkarkhatavkar** [comment]
> What's the benefit of having this test case versus the one we already use in ui automation? https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/ui/test_contentview.py#L2650

It is CLI test, Just wanted to ensure that content view filter rule, content view filter command should work as expected with module stream filter. If Any further change in API's could break this, what do you think?

File: tests/foreman/cli/test_contentview.py [hunk 1]
```diff
@@ -1231,6 +1233,57 @@ def test_positive_add_rh_repo_by_id_and_create_filter(self):
...
+        :expectedresults: Filter rule should get applied to Filter
+
+        :CaseImportance: Low
+
+        :CaseLevel: Integration
+
+        """
+        filter_name = gen_string('alpha')
+        repo_name = gen_string('alpha')
+        org = entities.Organization().create()
```
Comment (vijay8451) [inline tests/foreman/cli/test_contentview.py]: `setUpClass` is creating same :
https://github.com/SatelliteQE/robottelo/blob/6969daac216f48582eaf4c172fa528e8dc7a45f6/tests/foreman/cli/test_contentview.py#L171

File: tests/foreman/cli/test_contentview.py [hunk 2]
```diff
@@ -1231,6 +1233,57 @@ def test_positive_add_rh_repo_by_id_and_create_filter(self):
...
+            'type': 'modulemd',
+        })
+        content_view_filter_rule = ContentView.filter.rule.create({
+            'content-view-filter': filter_name,
+            'content-view-id': content_view['id'],
+            'module-stream-ids': walrus_stream['id'],
+        })
+        filter_info = ContentView.filter.info({'id': content_view_filter['filter-id']})
+        assert filter_info['rules'][0]['id'] == content_view_filter_rule['rule-id']
+
```
Comment (vijay8451) [inline tests/foreman/cli/test_contentview.py]: Just a thought , Do you think a content_view publish and check filtered packages in new version does add value here ?

Comment (omkarkhatavkar) [reply tests/foreman/cli/test_contentview.py]: https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/ui/test_contentview.py#L2650 test already doing it, I just wanted to see if the hammer commands work fine creating content view filter rules

**Comment by omkarkhatavkar** [comment]
> Personally I don't really think it's necessary if UI already covers this, then again I've seen many test cases that are in both. Since you've done the work, it's worthwhile to have it in place.
> 
> Ack

@latran Again as I mentioned in my comment https://github.com/SatelliteQE/robottelo/pull/7537#issuecomment-564678019 here I wanted to check if the hammer command works are expected for content filter rule. Any changes in the hammer code can not be covered by UI Test Automation.

**Pull request merged**